pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -580,6 +581,14 @@ type Scale struct {
 	// The is use to prevent too aggressive scaling operations
 	// +optional
 	ReplicasPerScale *uint32 `json:"replicasPerScale,omitempty" protobuf:"varint,9,opt,name=replicasPerScale"`
+	// ScaleUpCooldownSeconds defines the cooldown seconds after a scaling operation, before a follow-up scaling up.
+	// It defaults to the CooldownSeconds if not set.
+	// +optional
+	ScaleUpCooldownSeconds *uint32 `json:"scaleUpCooldownSeconds,omitempty" protobuf:"varint,11,opt,name=scaleUpCooldownSeconds"`
```

Comment:
Do we skip the number 10 on purpose and why?

Reply:
Ooops, Copilot generated.....